verify/verify.go

```diff
@@ -294,6 +295,10 @@ type CRLUnavailableErr struct {
 // GetCrlAndCheckRoot downloads the given cert's CRL from one of the distribution points and
 // verifies that the CRL is valid and doesn't revoke an intermediate key.
 func GetCrlAndCheckRoot(r *trust.AMDRootCerts, opts *Options) (*x509.RevocationList, error) {
+	return GetCrlAndCheckRootContext(context.Background(), r, opts)
+}
+
+func GetCrlAndCheckRootContext(ctx context.Context, r *trust.AMDRootCerts, opts *Options) (*x509.RevocationList, error) {
```

Comment:
Every exported function requires a documenting comment that starts with the function identifier

Reply:
Done